iOS Example Frame/iOS Example Frame/CustomStyle2.swift

```diff
@@ -24,9 +24,33 @@ private enum Constants {
 // MARK: - Payment Form
 // **********************
 
+struct PayButtonFormStyleCustom2: ElementButtonStyle {
+  var image: UIImage?
+  var text: String = "Pay 100$"
+  var textAlignment: NSTextAlignment = .center
+  var font: UIFont = UIFont.systemFont(ofSize: 15)
+  var disabledTextColor: UIColor = Constants.grayBackgroundColor
+  var disabledTintColor: UIColor =  Constants.grayBackgroundColor
+  var activeTintColor: UIColor =  Constants.greenBackgroundColor
+  var backgroundColor: UIColor = Constants.greenBackgroundColor
+  var textColor: UIColor = .white
```

Comment:
This button is invisible, is that intended?

Reply:
Same